pages/index.rst

```diff
@@ -70,13 +91,29 @@ The extensions that we are experimenting with include:
   defines a custom type (similar to ``ndarray`` but with many fewer features of
   course).
 
-- numpy-hpy_: one of our ambitious goals is to port numpy to HPy, and to use this
-  experience to better understand how to design the API. This is still at a
-  **very** early stage though.
+- numpy_: one of our ambitious goals is to port numpy to HPy, and to use this
+  experience to better understand how to design the API. This builds but does
+  not pass the test suite.
+
+- matplotlib_: Since Matplotlib also has a dependency to NumPy, the migration
+  to universal mode is not fully finished. HPy provides the legacy compatibility API
+  such that we can still call legacy C API functions from HPy and successfully
+  run the test suite.
+
+- kiwi-solver_: A dependency of Matplotlib, it has been fully ported to
+  universal mode.
+
+Benchmarks and more information about the Matplotlib and kiwi-solver ports can
+be found in the `GraalVM blog post`_. The benchmarks show that the HPy ports
+have little impact on CPython performance, while enabling close-to-cpython
+performance with GraalVM Python for the kiwi-solver complete HPy port.
 
 .. _ultrajson-hpy: https://github.com/hpyproject/ultrajson-hpy
 .. _piconumpy: https://github.com/hpyproject/piconumpy
-.. _numpy-hpy: https://github.com/hpyproject/numpy-hpy/
+.. _numpy: https://github.com/hpyproject/numpy-hpy/tree/graal-team/hpy#readme
+.. _matplotlib: https://github.com/hpyproject/matplotlib-hpy/
+.. _`kiwi-solver`: https://github.com/hpyproject/kiwi-hpy/
+.. _`GraalVM blog post`: https://medium.com/graalvm/porting-matplotlib-from-c-api-to-hpy-aa32faa1f0b5
 
```

Comment:
Would it be useful to also mention in this page what have been done on Cython? With a HPy backend, it will be possible to compile a lot of projects with universal extensions.

Reply:
I think the issue is that the Cython work is very early and can only compile a few basic tests right now. We could mention it, though.

Reply:
I agree. Is there more needed in this PR?

Reply:
I agree it is very early on, I wouldn't want to mention it yet.